packages/ui/package.json

```diff
@@ -5,7 +5,9 @@
   "types": "./index.tsx",
   "license": "MIT",
   "scripts": {
-    "build:icons": "node --experimental-modules internals/populate-icons.js"
+    "build:icons": "node --experimental-modules internals/populate-icons.js",
+    "storybook": "start-storybook -p 6006",
```

Comment:
add an alias for `dev` so that it gets started up on turbo repo run.

Reply:
im leaving for now. add it in if you want.